### DISCUSSION TITLE
Onboarding PR hasn't arrived

### DISCUSSION THREAD

**Discussion by paulgear** [renovatebot/renovate#27900] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

After enabling the app in our organisation and configuring it for a single repo, renovate appears in the app list and appears to be configured, but no onboarding PR has arrived, 45 minutes after initial configuration.  How long should we expect it to take?

Looking at the logs as recommended in the troubleshooting documentation, it seems that the renovate app might have run in dry run mode?
```
...
DEBUG: Creating onboarding branch
INFO: DRY-RUN: Would commit files to onboarding branch
DEBUG: Onboarding cache not updated
DEBUG: No renovate config file found
...
```
Log id is 85117552-68ef-48ae-bef5-1f938b39409c if any Mend staff are able to check it out.

(As an aside, the log interface is very difficult to read, being dark grey text on a black background.  It would be great if you could increase the contrast.)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
If you're seeing dry run mode then it means you installed the app with "silent" mode instead of "interactive". Check out the org/account Renovate settings and change it

<img width="901" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/cbe3ec7b-d3ef-49bc-b773-22e67134216a">

**Reply by paulgear** [reply]
Thanks very much for the prompt reply @rarkins!  That was it, and our onboarding PR has arrived now.  Is interactive mode not the default?  I can't think why our GH admins would have selected any non-default options.

**Reply by rarkins** [reply]
Silent mode is the default if you pick "All repositories"

**Reply by oscr** [reply]
@rarkins Thank you for this answer! :pray: Would you be ok with if I make a docs PR highlighting this, or e g adding a warning banner when in silent mode? I read and reread the `Installing and Onboarding` docs and could not understand why Renovate wasn't creating the onboarding PR.

**Reply by rarkins** [reply]
I thought there would be a banner in developer.mend.io on any repo which is in silent mode, is that not the case?

**Reply by oscr** [reply]
If there is a banner I didn't notice it when looking for the issue.

But I think more importantly it should be highlighted in the `installing-onboarding` docs, which has a screenshot with `All repositories` selected with the text `Renovate will ignore any repositories that don't have known package files, as well as any forks, so you can enable Renovate for all your repositories with no problems.`. Here I would like to add a warning about silent mode being activated and what it means.

**Reply by rarkins** [reply]
Thanks, I have passed on your feedback internally, we'll see what we can do to improve things for the next person!

**Reply by oscr** [reply]
Awesome! Thank you Rhys :pray:

**Reply by HonkingGoose** [reply]
I created an issue to improve the docs:

- https://github.com/renovatebot/renovate/issues/28154
